File: python/program/SivitasAkademik.py
from Human import Human


class SivitasAkademik(Human):
    def __init__(self, asal_universitas="", email_edu=""):  # constractor

        self.asal_universitas = asal_universitas
        self.email_edu = email_edu
# setter getter

    def set_asal_universitas(self, asal_universitas):
        self.asal_universitas = asal_universitas

    def set_email_edu(self, email_edu):
        self.email_edu = email_edu

    def set_prodi(self, prodi):
        self.prodi = prodi

    def get_asal_universitas(self):
        return self.asal_universitas

    def get_email_edu(self):
        return self.email_edu

File: CPP/PROGRAM/SivitasAkademik.cpp
#include "Human.cpp"
class SivitasAkademik : public Human
{
private: // private atribut
    string Asal_Universitas;
    string Email_edu;

public:
    SivitasAkademik() // CONSTRUCTOR
    {
    }

    void setAsal_Universitas(string b) // SET Asal_Universitas
    {
        Asal_Universitas = b;
    }
    void setEmail_edu(string c) // SET Email_edu
    {
        Email_edu = c;
    }

    string getAsal_Universitas() // GET Asal_Universitas
    {
        return Asal_Universitas;
    }
    string getEmail_edu() // GET Email_edu
    {

        return Email_edu;
    }

    ~SivitasAkademik() // DESTRUCTOR
    {
    }
};